src/modules/evaluateExpression/evaluateExpression.ts

```diff
@@ -79,17 +70,21 @@ export default async function evaluateExpression(
         if (Object.entries(params).length === 0)
           return 'No parameters received for objectProperties node'
         try {
-          return params[childrenResolved[0].object][childrenResolved[0].property]
+          const objectIndex = childrenResolved[0].objectIndex ? childrenResolved[0].objectIndex : 0
+          const inputObject = params && params.objects ? params.objects[objectIndex] : {}
+          const property = childrenResolved[0].property
```

Comment:
I haven't understood this line. Why you are just checking the `first` child?

Reply:
The `objectProperties` operator only contains one child node, with:
```{ objectIndex: <number>, property: <nameOfProperty> }```

I've tried to make it clear in the code by assigning the relevant items to the variable names `objectIndex`, `inputObject` and `property`.

Reply:
But yes, all the array indexing and nested object properties containing names of other properties does do one's head in. 🙄

Reply:
Ah. Ok, this is inside the `case 'objectParameters'`.